packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -1,639 +1,11 @@
-exports[`test should render a date picker with disabled dates 1`] = `
```

Comment:
Yeah, I'd recommend using `render` to render snapshots so we have more of the structure captured in the snapshot.